apps/minifront/src/state/staking/index.ts

```diff
@@ -129,20 +125,6 @@ export interface StakingSlice {
    * delegate or undelegate button for.
    */
   validatorInfo?: ValidatorInfo;
-  stakingTokensAndFilter: ZQueryState<{
-    /**
-     * A map of numeric account indexes to unstaked (UM) tokens for that account.
-     */
-    unstakedTokensByAccount: Map<number, ValueView | undefined>;
-    /**
-     * All accounts for which staking is relevant, to be passed to
-     * `<AccountSwitcher />` as the `filter` prop. This includes accounts with:
-     * - delegation tokens
-     * - unbonding tokens
-     * - staking (UM) tokens (since they can be delegated)
-     */
-    accountSwitcherFilter: number[];
-  }>;
```

Comment:
Now moved to the custom [`useStakingTokensAndFilter()` hook](https://github.com/penumbra-zone/web/pull/1339/files#diff-37aed1ca05297402c813cd650aab4035f8348b168d481cfa5333139abe42d0f7)